inference-engine/tests/ngraph_helpers/lpt_ngraph_functions/src/convolution_function.cpp

```diff
@@ -53,15 +58,36 @@ std::shared_ptr<ngraph::Function> ConvolutionFunction::getOriginal(
     convertOnWeights->constant_fold(convertedOutput, convertOnWeights->input_values());
     const auto convertedWeights = convertedOutput[0].get_node_shared_ptr();
 
-    const auto onWeights = fakeQuantizeOnWeights.empty() ? convertedWeights :
-        ngraph::builder::makeFakeQuantize(
-            convertedWeights, netPrecision,
-            fakeQuantizeOnWeights.quantizationLevel,
-            fakeQuantizeOnWeights.constantShape,
-            fakeQuantizeOnWeights.inputLowValues,
-            fakeQuantizeOnWeights.inputHighValues,
-            fakeQuantizeOnWeights.outputLowValues,
-            fakeQuantizeOnWeights.outputHighValues);
+    const std::shared_ptr<ngraph::Node> constant = std::make_shared<ngraph::opset1::Constant>(
+        ngraph::element::u64,
+        ngraph::Shape{4},
+        ngraph::Shape{0, 1, 2, 3});
```

Comment:
Minor: it seems strange because last parameter should be values, not shape. To my mind it would be better:
```suggestion
    const std::shared_ptr<ngraph::Node> constant = std::make_shared<ngraph::opset1::Constant>(
        ngraph::element::u64,
        ngraph::Shape{4},
        {0, 1, 2, 3});
```
P.S. Also, Constant has `create` method. So, we can write it shorter:
```suggestion
    const auto constant = ngraph::opset1::Constant::create(ngraph::element::u64, {4}, {0, 1, 2, 3});
```

Reply:
done, thanks